samples/csharp/getting-started/BinaryClassification_HeartDiseasePrediction/HeartDiseasePrediction-Solution/DataStructure/HeartData.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.ML.Data;
+
+namespace HeartDiseasePredictionConsoleApp.DataStructure
+{
+    public class HeartData
+    {
+        public float Age;
+        public float Sex;
+        public float Cp;
+        public float TrestBps;
+        public float Chol;
+        public float Fbs;
+        public float RestEcg;
+        public float Thalac;
+        public float Exang;
+        public float OldPeak;
+        public float Slope;
+        public float Ca;
+        public float Thal;
+    }
+
+    public class HeartDataImport
+    {
```

Comment:
You can use single class and speccify [LoadColumn()] attribute. No need of HeartImport class.